### ISSUE TITLE
Build fails because Git LFS for Debian Stretch requires Backports source

### ISSUE THREAD

**Issue by tjanson** [jenkinsci/docker-slave#60]
The `openjdk:8-jdk` based image does not build because the `git-lfs` package isn’t found in the default sources. It’s in Backports, so this needs to be added:

```
RUN echo "deb http://deb.debian.org/debian stretch-backports main" >> /etc/apt/sources.list
```

**Comment by tjanson** [comment]
In the `:11-jdk` this is [done here](https://github.com/docker-library/openjdk/blob/178c542fbb93a8f8a42e331b73a1214c9d8ba81d/11/jdk/Dockerfile#L25) as

```
RUN echo 'deb http://deb.debian.org/debian stretch-backports main' > /etc/apt/sources.list.d/stretch-backports.list
```

**Comment by oleg-nenashev** [comment]
Should be released as 3.29-2 in few hours
